src/main/java/teammates/common/datatransfer/FeedbackContributionQuestionDetails.java

```diff
@@ -576,8 +576,9 @@ public String getQuestionResultStatisticsCsv(
             FeedbackSessionResultsBundle bundle) {
         List<String> teamNames = new ArrayList<String>();
         for(FeedbackResponseAttributes response : responses){
-            if(!teamNames.contains(bundle.getTeamNameForEmail(response.giverEmail))){
-                teamNames.add(bundle.getTeamNameForEmail(response.giverEmail));
+            String teamNameOfResponseGiver = bundle.getUnsanitizedTeamNameForEmail(response.giverEmail);
+            if (!teamNames.contains(teamNameOfResponseGiver)) {
```

Comment:
Ok, I understood the meaning after reading the rest of the code. May you should have used a `Set` for this? Not related to this issue, so no need to change.